packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,555 @@
+### Components
+
+| Component         | Purpose                                                                  |
+| ----------------- | ------------------------------------------------------------------------ |
+| Breadcrumb        | Wrapper for the Breadcrumb component. Contains `nav` and `ol` elements.  |
+| BreadcrumbDivider | Divider component                                                        |
+| BreadcrumbItem    | `li` element. Can contain BreadcrumbButton or BreadcrumbLink components. |
+| BreadcrumbLink    | Breadcrumb Link                                                          |
+| BreadcrumbButton  | Breadcrumb Button                                                        |
+
+### Breadcrumb
+
+#### Anatomy
+
+![visual anatomy of the Breadcrumb component](./assets/breadcrumb-anatomy.png)
+
+#### DOM
+
+```HTML
+<nav aria-label="breadcrumb">
+  <ol>
+  {children}
+  </ol>
+</nav>
+```
+
+#### Breadcrumb structure:
+
+- root `nav` element
+- list `ol` element
+
+#### API
+
+| Property   | Values                                  | Default       | Purpose                                     |
+| ---------- | --------------------------------------- | ------------- | ------------------------------------------- |
+| appearance | `transparent`, `subtle`                 | `transparent` | Sets appearance                             |
+| focusMode  | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`         | Sets the focus behavior for the Breadcrumb. |
+| size       | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb              |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button - BreadcrumbButton component is used inside BreadcrumbItem.
+- Link - BreadcrumbLink is used inside BreadcrumbItem.
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbButton onClick={() => console.log('smth...')}>
+    Item 2
+  </BreadcrumbButton>
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbLink href="#">
+    Item 3
+  <BreadcrumbLink>
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default  | Purpose                                     |
+| ------------ | --------------------------------------- | -------- | ------------------------------------------- |
+| active       | boolean                                 | false    | Indicates if the item is active             |
+| disabled     | boolean                                 | false    | Disables Breadcrumb item                    |
```

Comment:
Are these props applicable to `BreadcrumbItem`? It feels that they only have sense for `BreadcrumbButton` & `BreadcrumbLink`.

BTW, I checked the spec in Figma and it seems that "active" is just a `:active` pseudo class.

Reply:
There's `disable` state for non-interactive item in the design and it, also, have `active` state. Thugh there shoudn't be `disabled` for non-interactive items. I'll double-check with the designers